citi-onboarding/acervusej-onepage
# Fix/finalchanges

Things I've done:

 Fix the font-sizes on the front end
 Delete Post model, controller and route on back-end
 Delete Post component on front-end
 Create a base_url for axios
 Update axios urls

How to test it:

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write http://localhost:3000/.
- See if everything works both in mobile and desktop view.
